dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/async/WrapperRunnableImpl.java

```diff
@@ -104,8 +94,6 @@ public void run() {
         }
         // End:PM90834
 
-        SRTServletRequestThreadData.getInstance().init(null);
```

Comment:
Somehow srtRequestDataInitNull() logic needs to happen which is why this was done as a thread local because I don't think there is a reference to the SRTServletRequest here.  If there is then srtRequestDataInitNull needs to be called.